be/src/util/file_utils.h

```diff
@@ -80,7 +106,11 @@ class FileUtils {
     // calc md5sum of a local file
     static Status md5sum(const std::string& file, std::string* md5sum);
 
+    // check path(file or directory) exist with default env
     static bool check_exist(const std::string& path);
+    
+    // check path(file or directory) exist with env
+    static bool check_exist(const std::string& path, Env* env);
```

Comment:
I think you should refactor this function to make it return error if status is no NOT_FOUND.
And this may lead many changes, you can resolve this in another ISSUE

Reply:
Recorded ISSUE #2166. Will modify later